vue-project-stub-no-pwa/src/interfaces/resource-interface.ts

```diff
@@ -0,0 +1,6 @@
+// Interface to define the resource model example
+export interface ResourceExampleIF {
+  id: number,
```

Comment:
Commas or semi-colons at end of line? Or neither? I dunno, pick something and we'll try to stick to it.

Reply:
Future